### PR TITLE
Added free(fifon); to some lines for memory leak

### PR DESCRIPTION
**Pull request by KocogluFerdi**

### CODE REVIEW

**Review by stevenvisla-unikie — DISMISSED** [review]
I didn't run the code on VM, but changes are logical and valid to me 👍

**Review by stevenvisla-unikie — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -66,4 +73,4 @@ On ubuntu these can be installed with: `sudo apt install xxd openssl libssl-dev`
 ## Warning
 The program **must** be built with meson as it generates the header file for the help 
 message, if not using meson then an equivalent header file containing the `___src_help_str_txt_len`
-and ` ___src_help_str_txt` variables must be generated
```
Comment (antonkraft) [inline README.md:69]: Just curious what was changed here

**Review by antonkraft — APPROVED** [review]
Not very exiting changes to look at. Naturally no issues found. Of course I have no knowledge about the functionality of the files, that I have not seen during my last review (mostly pipe) and judged the changes individually.

File: README.md (hunk 1 above)
Comment (LorenzoRuffati) [reply README.md:69]: I think just the diff algorithm getting confused

**Review by LorenzoRuffati — APPROVED** [review]

**Pull request merged**